aiogram/types/chat.py

```diff
@@ -512,6 +513,7 @@ def _check(obj, chat_types) -> bool:
         return obj.type in chat_types
 
     @classmethod
+    @deprecated("This filter was moved to ChatTypeFilter, and will be removed in aiogram v3.0")
```

Comment:
What's the motivation of deprecating those `is_{type}` properties? I think we should just leave these features because they may remain useful as we don't get the result of filter's `check` in your current implementation.

Reply:
As far as I understood the context of this feature by @JrooTJunior [here](https://github.com/aiogram/aiogram/issues/151#issuecomment-513668019), current filters should be marked as deprecated.

By the way, `ChatType.is_private(obj)` is the same as `ChatTypeFilter(chat_types[ChatType.PRIVATE]).check(obj)`